Source/Database.cpp

```diff
@@ -74,7 +74,7 @@ Game Database::getGameById(unsigned int id)
     return {id, name, path, exe};
 }
 
-Game Database::getGameByName(QString name)
+Game* Database::getGameByName(QString name)
```

Comment:
We can take it out, it was just a check to verify that the game you're adding isn't already in there. It doesn't break anything, and a better way should be found IMO.

Are you fine with taking it out?